python/dgl/mock_sparse/sp_matrix.py

```diff
@@ -171,7 +171,7 @@ def __call__(self, x):
         assert len(x) == self.nnz
         return SparseMatrix(self.row, self.col, x, shape=self.shape)
 
-    def indices(self, fmt, return_shuffle=False) -> Tuple[torch.tensor, ...]:
+    def indices(self, fmt:str, return_shuffle=False) -> Tuple[torch.Tensor, ...]:
```

Comment:
Are we preferring `fmt:str` or `fmt : str` or `fmt: str`? cc @frozenbugs

Reply:
fmt: str
https://docs.python.org/3/library/typing.html

Reply:
black will take care of this.